Source/NSubstitute.Acceptance.Specs/FieldReports/Issue237_ReceivedInOrderErrorHandling.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using NUnit.Framework;
+
+namespace NSubstitute.Acceptance.Specs.FieldReports
+{
+    public class Issue237_ReceivedInOrderErrorHandling
+    {
+        public interface IAmAnInterface
+        {
+            void MethodA(int a, int b);
+            void MethodB(int a, int b);
+
+            int MethodC();
+        }
+
+        [Test]
+        [ExpectedException(typeof(Exception))]
+        public void AnExceptionIsReceivedWhenExpected()
+        {
+            Received.InOrder(() =>
+            {
+                throw new Exception("An Exception!");
+            });
+        }
+
+        [Test]
+        public void MethodCallsAreReceivedInOrder()
+        {
+            IAmAnInterface _interface = Substitute.For<IAmAnInterface>();
+            _interface.MethodA(1, 2);
+            _interface.MethodB(1, 2);
+            Received.InOrder(() =>
+            {
+                _interface.MethodA(1, 2);
+                _interface.MethodB(1, 2);
+            });
+        }
+
+        [Test]
+        public void MethodCallsAreReceivedInOrder2()
```

Comment:
What's the reason to have two identical tests: MethodCallsAreReceivedInOrder and this one?